src/clone.rs

```diff
@@ -223,12 +227,24 @@ macro_rules! to_return_value {
 ///     fn foo(&self) {
 ///         let v = Rc::new(1);
 ///
-///         let closure = clone!(self => move |x| {
+///         let closure = clone!(@strong self => move |x| {
 ///             println!("v: {}, x: {}", v, x);
 ///         });
 ///     }
 /// }
 /// ```
+///
+/// **Using another keyword than `weak` or `strong`**:
```

Comment:
Why do you put an example in the docs for this? :) Nobody should ever do that and having that in the docs only distracts really.